func/func_server.cpp

```diff
@@ -0,0 +1,30 @@
+#include <func_server.h>
+
+  Status FuncServiceImpl::hook(ServerContext* context, const HookRequest* request, HookReply* reply) {
+    // TODO
+  }
+
+  Status FuncServiceImpl::unhook(ServerContext* context, const UnhookRequest* request, UnhookReply* reply){
+    // TODO:
+  }
+
+  Status FuncServiceImpl::event(ServerContext* context, EventRequest* request, EventReply* reply){
+    //  TODO: 
+  }
+
```

Comment:
the indentation is off on these
Also fill in the todo's with actual things you need to do

Reply:
changed

Reply:
Close, though you should have these functions be aligned with zero spaces.

Reply:
Forgot to push, sorry